src/scancode/cli.py

```diff
@@ -109,6 +113,13 @@
 
 notice = acknowledgment_text.strip().replace('  ', '')
 
+scan_proper_plugins = PluginManager('scan_proper')
+scan_output_plugins = PluginManager('scan_output')
```

Comment:
My hunch is that the PluginManager should get you all you need to validate the formats are unique and more? e.g. I feel this is what you should use and not the `plugin_formats` global custom mapping